### ISSUE TITLE
No syntax and highlight when using template string with string interpolation

### ISSUE THREAD

**Issue by xristosNSK** [styled-components/vscode-styled-components#395]
I'm currently using `@mui/material` which includes `emotion/react`. The following syntax doesn't seem to work.

![image](https://user-images.githubusercontent.com/48767092/209238228-96faff91-f8f9-48bc-9ebb-f7619e29c300.png)

Any suggestions?

**Comment by oscarvz** [comment]
Just like with `styled-components`, return the `css` template string function instead of a plain template literal. This'll not only fix syntax highlighting, but also CSS linters in case you're using those 😄 

```tsx
import { css, styled } from '@mui/material';

export const PropertyWrapper = styled('div')(
  ({ theme: { palette } }) => css`
    border-radius: 25px;
    border: 1px solid ${palette.divider};
  `
);

```

**Comment by Twipped** [comment]
@oscarvz I don't believe this works the way you think it does. When I tried it, I got errors about trying to stringify the object that MUI's `css` function produces.

**Comment by oscarvz** [comment]
@Twipped [this sandbox](https://codesandbox.io/s/goofy-dan-517igl?file=%2Fsrc%2FApp.tsx%3A0-23) demonstrates the difference in syntax highlighting. I've been using this approach for basically every project (albeit mostly with `styled-components` and only once with MUI) as otherwise linting & syntax highlighting won't work.

Do those errors you mention pop up when building your project?

**Comment by Twipped** [comment]
@oscarvz they were popping up when running jest tests, so it may actually be an issue with emotion's interactions with JSDOM

**Comment by aleksandrjet** [comment]
Can I use this functional with `styled.div` instead `styled('div')`? I use import from `@emotion/styled`.

It is wished variant of usage, but I don't see syntax highlight in Header component:
<img width="560" alt="Screenshot 2024-04-12 at 14 25 41" src="https://github.com/styled-components/vscode-styled-components/assets/79018733/a6be19f6-59c2-4e56-a54a-8ec71f2cd9c3">

When I try to use import from `@mui/material` I get error, that "Property 'div' does not exist on type 'CreateMUIStyled<Theme>'":
<img width="563" alt="Screenshot 2024-04-12 at 14 26 24" src="https://github.com/styled-components/vscode-styled-components/assets/79018733/f4952e26-2a3f-4570-a63a-0b3912640fe4">

I can fix it, if I use import from emotion and material at the same time:
<img width="560" alt="Screenshot 2024-04-12 at 14 29 59" src="https://github.com/styled-components/vscode-styled-components/assets/79018733/9675a00c-7a91-4a6a-be5e-d6f796d572c2">

Do it have any methods of usage to fix it without extra imports?
